msal/tests/Test.MSAL.NET.UIAutomation/MSALMobileTestHelper.cs

```diff
@@ -140,5 +179,24 @@ private void ValidateUiBehaviorString(string uiBehavior)
                 throw new InvalidOperationException("Test Setup Error: invalid uiBehavior " + uiBehavior);
             }
         }
+
+        public void PerformB2CLocalAccountSignInFlow(ITestController controller, LabUser user)
+        {
+            UserInformationFieldIds userInformationFieldIds = CoreMobileTestHelper.DetermineUserInformationFieldIds(user);
+
+            controller.Tap(CoreUiTestConstants.AcquirePageID);
+
+            //Acquire token flow
+            controller.Tap(CoreUiTestConstants.AcquireTokenID);
+
+            //id="logonIdentifier" for local b2c accounts
+            controller.EnterText(CoreUiTestConstants.WebUPNB2CLocalInputID, 20, user.Upn, XamarinSelector.ByHtmlIdAttribute);
+
+            //id="password"
+            controller.EnterText(userInformationFieldIds.PasswordInputId, LabUserHelper.GetUserPassword(user), XamarinSelector.ByHtmlIdAttribute);
+
```

Comment:
nit: comments are not needed

Reply:
done.

---
In reply to: [233289201](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r233289201) [](ancestors = 233289201)